### ISSUE TITLE
Wrong error message when there is an error in d.ts file

### ISSUE THREAD

**Issue by stereobooster** [microsoft/TypeScript#33530]
**TypeScript Version:**  3.6.2

**Search Terms:** "debug d.ts"

**Code**

```ts
import * as Leaflet from "leaflet";

declare module "leaflet-freedraw" {
  export default any;
  export const ALL: number;
  export const APPEND: number;
  export const CREATE: number;
  export const DELETE: number;
  export const EDIT: number;
  export const EDIT_APPEND: number;
  export const NONE: number;
  export const edgesKey: any;
  export const instanceKey: any;
  export const modesKey: any;
  export const notifyDeferredKey: any;
  type DefaultFreeDrawOptions = {
    concavePolygon: boolean;
    elbowDistance: number;
    leaveModeAfterCreate: boolean;
    maximumPolygons: number;
    mergePolygons: boolean;
    mode: number;
    notifyAfterEditExit: boolean;
    simplifyFactor: number;
    smoothFactor: number;
    strokeWidth: number;
  };
  export type FreeDrawOptions = Leaflet.LayerOptions & Optional<DefaultFreeDrawOptions>;
  declare class FreeDraw extends Leaflet.Layer {
    constructor(opts: FreeDrawOptions);
    mode: number;
  }
  export const defaultOptions: DefaultFreeDrawOptions;
  export default FreeDraw;
  namespace polygons {
    function deleteAll(): any;
    function get(p0: any): any;
    function has(p0: any): any;
    function set(p0: any, p1: any): any;
  }
}
```

**Expected behavior:** 

I'm adding "local" types to untyped library. When there is an error in d.ts file it reports it as 

```
TS7016: Could not find a declaration file for module 'leaflet-freedraw'. '/node_modules/leaflet-freedraw/dist/leaflet-freedraw.cjs.js' implicitly has an 'any' type.
```

Instead it would be nice to see what the actual error. (How to debug d.ts files?)

If I change file to:

```ts
declare module "leaflet-freedraw";
```

It reports no error.

**Actual behavior:** reports unrelated error

```
TS7016: Could not find a declaration file for module 'leaflet-freedraw'. '/node_modules/leaflet-freedraw/dist/leaflet-freedraw.cjs.js' implicitly has an 'any' type.
```

**Playground Link:**  N/A because I have no idea how to reproduce it in playground

**Related Issues:** No

**Comment by stereobooster** [comment]
`"skipLibCheck": false,` is what I needed. How I missed it?
